trillian/ctfe/ct_server/main.go

```diff
@@ -123,7 +122,11 @@ func main() {
 		}()
 	} else {
 		// Use a fixed endpoint resolution that just returns the addresses configured on the command line.
```

Comment:
Comment is now incorrect.

Reply:
I thought it still applies - the endpoints are still fixed, and it'll still return the addresses of the DNS names passed in on the command line...

Reply:
Arguably it's not a fixed resolution though.

Reply:
removed the "fixed" bit, PTAL.